addons/mass_mailing/views/mailing_mailing_views.xml

```diff
@@ -539,31 +532,13 @@
                                             </div>
                                         </div>
                                     </div>
-                                    <div class="o_kanban_record_body" t-if="!selection_mode" invisible="not sent_date and not schedule_date and state != 'in_queue'">
-                                        <div>
-                                            <span invisible="not sent_date" class="me-1"><b><field name="delivered"/> / <field name="expected"/></b> Delivered to</span>
-                                            <span invisible="sent_date" class="me-1"><b><field name='total'/></b></span>
-                                            <field name='mailing_model_id' invisible="mailing_on_mailing_list"/>
-                                            <span invisible="not mailing_on_mailing_list">Mailing Contact</span>
-                                        </div>
-                                        <div invisible="not sent_date" class="d-flex justify-content-between">
-                                            <div name="stat_opened">
-                                                <b><field name="opened_ratio" />%</b> Opened
-                                            </div>
-                                            <div name="stat_replied">
-                                                <b><field name="replied_ratio" />%</b> Replied
-                                            </div>
-                                            <div name="stat_clicks">
-                                                <b><field name="clicks_ratio" />%</b> Clicks
-                                            </div>
-                                        </div>
-                                    </div>
                                 </div>
+                                <i class="fa fa-bullseye" role="img" aria-label="Lead/Opportunity"> Lead/Opportunity</i> <br />
```

Comment:
Suggestion to keep the correct font and add a margin to align this part with the line below
```suggestion
                                <i class="fa fa-bullseye me-2" role="img" aria-label="Lead/Opportunity"/> Lead/Opportunity
```